google_guest_agent/oslogin.go

```diff
@@ -99,15 +99,20 @@ func (o *osloginMgr) set(ctx context.Context) error {
 		logger.Infof("Disabling OS Login")
 	}
 
-	if err := writeSSHConfig(enable, twofactor, skey); err != nil {
+	// [Unstable] configuration section has no o long term stability or support guarantees/promisses.
```

Comment:
Nice, this is much clearer. Thank you.

A couple of typo fixes on this line; it should read:

    [Unstable] configuration section has no long term stability or support guarantees/promises.